android/app/src/main/java/app/organicmaps/maplayer/MapButtonsController.java

```diff
@@ -193,6 +196,29 @@ private static int dpToPx(float dp, Context context)
     return (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dp, context.getResources().getDisplayMetrics());
   }
 
+  @OptIn(markerClass = ExperimentalBadgeUtils.class)
```

Comment:
What are the pros of using experimental code?

Reply:
This method was pre implemented and these experimental badge features too. I have just created one more similar method with arguments. 

I guess this is necessary here since we are dealing with badge which is experimental probably.

Please have a look into updateBadge() method in current master branch. This is already used there to show badges.